app/scripts/views/base.js

```diff
@@ -389,6 +389,29 @@ define(function (require, exports, module) {
 
     // called after the view is visible.
     afterVisible: function () {
+      // restyle side-by-side links to stack if they are too long
+      // to fit on one line
+      var linkContainer = this.$el.find('.links');
+      if (linkContainer.length > 0) {
+        var containerWidth = linkContainer.width();
+        var halfContainerWidth = containerWidth / 2;
+        var shouldResetLinkSize = false;
+
+        linkContainer.children().each(function (i, item) {
```

Comment:
to avoid extra work here can we look only for **links** (<a ... ) in the container children? (.children() has an optional [selector] option)